plugins/kafka-info/src/components/KafkaInfoComponent/KafkaInfoComponent.tsx

```diff
@@ -17,6 +22,21 @@ import { useEntity } from '@backstage/plugin-catalog-react';
 // These will let us get info about our backstage configuration
 import { useApi, configApiRef } from '@backstage/core-plugin-api';
 
+const clusterMap = [
+  stage: {
```

Comment:
I'm getting syntax failures here, not sure if it's my dev env or something valid. It doesn't seem to be liking named objects within the array?

Working through a couple of other errors in my env, but this seems to be the major break for me. @adamrdrew seeing similar?